meshtastic/node_runtime/content_runtime.py

```
@@ -0,0 +1,350 @@
+        admin_message = decoded.get("admin")
+        if not isinstance(admin_message, dict):
+            logger.warning("Unexpected ringtone response without admin payload")
+            return False
+        raw_admin = admin_message.get("raw")
+        if raw_admin is None or not hasattr(raw_admin, "get_ringtone_response"):
+            logger.warning("Unexpected ringtone response without raw ringtone data")
+            return False
+        try:
+            ringtone_part = raw_admin.get_ringtone_response
+            self._cache_store.store_ringtone_fragment(ringtone_part)
+            return True
```

Ruff (0.15.6)

[warning] Consider moving this statement to an `else` block

(TRY300)